src/navigator-card-body.jsx

```diff
@@ -75,7 +82,102 @@
     }
 };
 
+// eslint-disable-next-line max-len
+const ContextMenuItems = ({ path, currentDir, selected, setSelected, setHistory, setHistoryIndex, addAlert, clipboard, setClipboard, files }) => {
+    const Dialogs = useDialogs();
+
+    const _createDirectory = () => createDirectory(Dialogs, currentDir);
+    const _createLink = () => createLink(Dialogs, currentDir, files, selected[0]);
+    const _copyItem = () => {
+        copyItem(setClipboard, selected.length > 1
+            ? selected.map(s => currentDir + s.name)
+            : [currentDir + selected[0].name]);
+    };
+    const _pasteItem = (targetPath, asSymlink) => pasteItem(clipboard, targetPath.join("/") + "/", asSymlink, addAlert);
+    const _renameItem = () => renameItem(Dialogs, { selected: selected[0], path, setHistory, setHistoryIndex });
+    const _editPermissions = () => editPermissions(Dialogs, { selected: selected[0], path });
+    const _deleteItem = () => {
+        deleteItem(
+            Dialogs,
+            {
+                selected,
+                setHistory,
+                setHistoryIndex,
+                path: currentDir,
+                setSelected
+            }
+        );
+    };
+
+    if (selected.length === 0)
+        return null;
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-237-20240201-172301-e0f83b57-fedora-39-devel//Coverage/lcov/github-pr.diff.gcov.html)